components/datepicker/stories/datepicker.stories.js

```diff
@@ -63,15 +62,6 @@ export default {
 			},
 			control: "boolean",
 		},
-		isRequired: {
```

Comment:
We don't need isValid or isRequired anymore?

Reply:
At the time I was thinking we didn't since there is no CSS written for these states with in DatePicker. I do see now that there are slight HTML changes.